doc/UsersGuide/source/simulationflags.rst

```diff
@@ -6,6 +6,63 @@ as well as additional details of the :ref:`numerical integration methods <crunti
 
 .. _cruntime-simflags :
 
+
+Repetition of Simulation Flags and Options
+-------------------------------------------------
+Unless it is explicitly specified otherwise, it **is an error** to specify any of the simulation Flags/Options more than once.
+
+
+Repetition policies for specific flags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If a Flag/Option is allowed to be repeated, there are three different policies under which it can be processed.
+
+- **Ignore**: Repeated instances of the Flag/Option will be ignored with a warning. For example
+
+    .. code-block:: bash
+
+       $ sim_exe -cx=file1 -cx=file2
+       $ sim_exe -cx=file1
+
+    are equivalent except for the warning issued in the first case.
+
+- **Replace**: Repeated instances of an option will override the current value. (Not applicable for Flags.)
+
+    .. code-block:: bash
+
+       $ sim_exe -cx=file1 -cx=file2
+       $ sim_exe -cx=file2
+
+    are equivalent except for the warning issued in the first case.
+
+
+- **Combine**: Repeated instances of the option's values will be combined as if they were given as values for a single specification of the option. (Not applicable for Flags.)
+
+    .. code-block:: bash
+
+       $ sim_exe -cx=file1 -cx=file2
+       $ sim_exe -cx=file1,file2
+
+    are equivalent except for the warning issued in the first case. Assuming the option supports multiple values.
+
+
+Please check the entry below (or the help messages) for each flag/option to find out if a it is allowed to be repeated and what the specific repetition rules are for it.
+
+Order of flags/options from OpenModelica tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The final order of flag/option specifications are dependent on where they are specified.
+
+If you are running the simulation executable directly then you have complete control of how the flags are ordered as it will be exactly the way you pass them on the command line.
```

Comment:
Are you sure about this? It would make sense to me that flags specified in vendor annotations are taken into account also if the simulation executable is run directly, so in this case you'd just skip point 2 below.

Do I miss something?

Reply:
Hmm. I think we need @adrpo here. As far as I can remember the flags are never embedded into the executable itself. It is just that, when you used OMEdit for example,  the executable is launched with a script (shell or cmd, not Modelica) that adds the flags to the command line.

I might be wrong though.

Reply:
NVM now that I remember, the executable is launched directly by omc itself. No script involved. I will go find the code and check how the flags are passed to the exe.

Reply:
> NVM now that I remember, the executable is launched directly by omc itself.
 No script involved. 

That's what I understood. Hence my guess that in this case we simply skip step 2.

Reply:
I still do not think the flags are embedded in the executable :).  Maybe they are in the init files. I will check properly.

Reply:
Sure, I have no idea where they are actually stored. My point is that I see no reason or rationale why flags set by annotations should be ignored when running the executable directly.

Reply:
check out https://github.com/OpenModelica/OpenModelica/pull/7828#issuecomment-911587271 to see what is happening currently.

I think point 2 might be corrected albeit for not the originally assumed behavior. What I have realized now is point 3 does not make much sense. You can not specify `simFlags` on the command line if you are running a script. So the point is unnecessary and confusing.